modules/order/src/Plugin/Field/FieldFormatter/OrderTotalSummary.php

```diff
@@ -76,10 +76,15 @@ public static function create(ContainerInterface $container, array $configuratio
    */
   public function viewElements(FieldItemListInterface $items, $langcode) {
     $order = $items->getEntity();
-    return [
-      '#theme' => 'commerce_order_total_summary',
-      '#totals' => $this->orderTotalSummary->buildTotals($order),
-    ];
+
+    $element = [];
+    foreach ($items as $delta => $item) {
```

Comment:
This doesn't make sense, we'd just repeat the same summary for every delta. Let's just hardcode the delta 0, since we know that a price field can't have more than 1.